corehq/apps/users/models.py

```diff
@@ -2001,6 +2007,39 @@ def get_location_map_case(self):
         except ResourceNotFound:
             return None
 
+    @property
+    @skippable_quickcache([], lambda _: settings.UNIT_TESTING)
```

Comment:
does the `quickcache` `vary_on` automatically include `self`? I don't think I've seen it used like this before and am worried this is actually caching a singleton when you want an instance level cache.
cc @dannyroberts